etc/production.bash

```diff
@@ -138,6 +138,14 @@ tar xf libs.tar
 cp ${etc}/emotiq.bash.${kind} ${production_dir}/emotiq.bash
 cd ${lib}
 rm *
-# < /test >
 
-# cp ${???}/*.so* ${production_dir}
+cd ${production_dir}
+case ${kind} in
+    linux) 
+	tar cfj ${kind}.emotiq.bz2 em* *.so*
```

Comment:
It is unexpected for tar archives to untar their contents into the current directory. 

It is more usual for a given  archive to contain their contents contained in top-level directory within the archive.  

so an untar of `emotiq-20180420a-linux.tar.bz2` would produce
```
./emotiq-20180420a-linux/emotiq
./emotiq-20180420a-linux/emotiq.bash
./emotiq-20180420a-linux/libgmp.so
…
```